packages/@icon-magic/generate/src/plugins/svg-to-raster.ts

```diff
@@ -49,13 +49,24 @@ export const svgToRaster: GeneratePlugin = {
       let h: number;
       // if a nameSizeMapping should be used, get the size from the matching name pattern
       if (params.useNameSizeMapping) {
+        const nameSizeMapping: { [name: string]: AssetSize } =
```

Comment:
Oddly, no. hmm...

Reply:
i believe this is because `nameSizeMapping` ends up being an `any` according to the `MetaData` interface. Would be good to better define that in the `MetaData` interface

Reply:
Done.